packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/blob/FileReaderModule.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.modules.blob
+
+import android.util.Base64
+import com.facebook.fbreact.specs.NativeFileReaderModuleSpec
+import com.facebook.infer.annotation.Nullsafe
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.module.annotations.ReactModule
+
+@Nullsafe(Nullsafe.Mode.LOCAL)
+@ReactModule(name = NativeFileReaderModuleSpec.NAME)
+public class FileReaderModule(reactContext: ReactApplicationContext) :
+  NativeFileReaderModuleSpec(reactContext) {
+
+  public companion object {
```

Comment:
We can move the companion object at the bottom of the file, a bit more Kotlin idiomatic